com.unity.render-pipelines.core/Runtime/Lights/LightAnchor.cs

```diff
@@ -10,17 +10,17 @@ namespace UnityEngine
     /// Represents camera-space light controls around a virtual pivot point.
     /// </summary>
     [AddComponentMenu("Rendering/Light Anchor")]
-    [RequireComponent(typeof(Light))]
     [ExecuteInEditMode]
     [DisallowMultipleComponent]
     [CoreRPHelpURLAttribute("View-Lighting-Tool")]
     public class LightAnchor : MonoBehaviour
     {
+        const float minDistance = 0.001f;
```

Comment:
In convension, constant should be prefixed with k_
```suggestion
        const float k_MinDistance = 0.001f;
```